source/numerics/error_estimator_inst2.cc

```diff
@@ -0,0 +1,20 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2015 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+// This file compiles the second half of the instantiations from error_estimator.cc
+// to get the memory consumption below 1.5gb with gcc.
+
+#define SPLIT_INDEX 1
+#include "error_estimator.cc"
```

Comment:
I would love to do this for fe_values.cc, but this doesn't work with the weird trick we do with fe_values.decl.1.inst (generate overloaded virtual functions for each vector type by abusing the .inst mechanism). Do you have a suggestion on how to handle this without including it inside the new fe_values.templates.h?